packages/manager/modules/emailpro/src/domain/emailpro-domain.html

```diff
@@ -106,22 +106,29 @@
                         <tr>
                             <td class="text-center" colspan="6">
                                 <div data-ng-if="!search.value">
-                                    <span
-                                        data-translate="{{:: exchange.isMXPlan
-                                        ? 'emailpro-mxplan_tab_domain_no_domains_1'
-                                        : 'emailpro_tab_domain_no_domains_1'
-                                    }}"
-                                    ></span>
-                                    <button
-                                        class="btn btn-link p-0"
-                                        type="button"
-                                        style="font-size:inherit"
-                                        data-translate="emailpro_tab_domain_no_domains_2"
-                                        data-ng-click="setAction('emailpro/domain/add/emailpro-domain-add')"
-                                    ></button>
-                                    <span
-                                        data-translate="emailpro_tab_domain_no_domains_3"
-                                    ></span>
+                                    <div data-ng-if="canAddDomain">
+                                        <span
+                                            data-translate="{{:: exchange.isMXPlan
```

Comment:
```suggestion
                                            data-translate="{{:: 
```
can be removed I think